pages/solid-events.md

```diff
@@ -6,16 +6,25 @@ permalink: /events
+Solid Events are an opportunity for people who are working on Solid or are interested in Solid to meet. Solid Events are run by people like you. If you are interested in coordinating a Solid Event near you, reach out to the Solid Manager at [info@solidproject.org](mailto:info@solidproject.org).
 
-In light of the COVID-19 pandemic Solid Events are moving online for the foreseeable future. Tune in on the first Thursday of every month at 1600 CET to join.
 
-Are you interested in speaking about what you have been building on Solid? Reach out to the Solid Manager on [info@solidproject.org](mailto:info@solidproject.org) to get a speaking slot at the next event.
+{:.has-background-light .has-text-danger}
+> In light of the COVID-19 pandemic, Solid Events are moving online for the foreseeable future. Join Solid World on the first Thursday of every month at 1600 CET.
+
+Are you interested in speaking about what you have been building on Solid? Reach out to the Solid Manager at [info@solidproject.org](mailto:info@solidproject.org).
+
+## 2021 Events
 
 | Date | Event | Organiser | Recordings, Slides and Notes |
 |----|-------|-------------|------------------------------|
 | 2021-02-04 | [Solid World February](https://www.eventbrite.com/e/solid-world-tickets-135541896461)|[Solid Team](https://solidproject.org/team)||
-| 2021-01-07 | [Solid World January](https://www.eventbrite.com/e/solid-world-tickets-132301957717)|[Solid Team](https://solidproject.org/team)|[Recording](https://vimeo.com/498076263).  Resources from event: [Solid Operating System (OS) Roadmap](https://solidos.solidcommunity.net/public/Roadmap/Tasks/index.ttl#this), [Community Solid Server (CSS)](https://solid.github.io/community-server/docs/), [Inrupt's Enterprise Solid Server (ESS)](https://inrupt.com/products/enterprise-solid-server), [Ruben's Code for the lights](https://github.com/RubenVerborgh/solid-hue), [Specifications](https://solid.github.io/specification/#work-items), [Specifications Panels (open to all)](https://github.com/solid/process/blob/master/panels.md).|
+| 2021-01-07 | [Solid World January](https://www.eventbrite.com/e/solid-world-tickets-132301957717)|[Solid Team](https://solidproject.org/team)|[Recording](https://vimeo.com/498076263).  <br>Resources from event: [Solid Operating System (OS) Roadmap](https://solidos.solidcommunity.net/public/Roadmap/Tasks/index.ttl#this), [Community Solid Server (CSS)](https://solid.github.io/community-server/docs/), [Inrupt's Enterprise Solid Server (ESS)](https://inrupt.com/products/enterprise-solid-server), [Ruben's Code for the lights](https://github.com/RubenVerborgh/solid-hue), [Specifications](https://solid.github.io/specification/#work-items), [Specifications Panels (open to all)](https://github.com/solid/process/blob/master/panels.md).|
```

Comment:
Later on, we may want to use something other than a comma for a list of links, but didn't want to be too pushy with this small pull 😄